src/Autofac.Extras.MvvmCross/AutofacMvxIocProvider.cs

```diff
@@ -47,18 +48,34 @@ public class AutofacMvxIocProvider : MvxSingleton<IMvxIoCProvider>, IMvxIoCProvi
         /// <param name="container">
         /// The container from which dependencies should be resolved.
         /// </param>
+        /// <param name="propertyInjectionEnabled"></param>
         /// <exception cref="System.ArgumentNullException">
         /// Thrown if <paramref name="container"/> is <see langword="null"/>.
         /// </exception>
-        public AutofacMvxIocProvider(IContainer container)
+        public AutofacMvxIocProvider(IContainer container, bool propertyInjectionEnabled)
```

Comment:
To avoid a breaking interface change, we need to add a second constructor - one should just take the container (like it used to) and the other can take the Boolean parameter. I'm guessing the first constructor should just call the second one with "false" as the parameter.